iothub/device/src/Messaging/TelemetryMessage.cs

```diff
@@ -29,6 +29,15 @@ public TelemetryMessage(object payload)
             Payload = payload;
         }
 
+        /// <summary>
+        /// Creates an instance of this class with the specified binary payload.
+        /// </summary>
+        /// <param name="binaryPayload">The binary payload to send.</param>
+        public TelemetryMessage(byte[] binaryPayload)
```

Comment:
Does not yield an ambiguity compiler error as expected, so we offer this overload for clarity.

@rido-min